src/main/java/gift/kakao/KakaoMessageInterface.java

```diff
@@ -13,14 +13,14 @@
 @HttpExchange()
 public interface KakaoMessageInterface {
 
-    @PostExchange(value = "${custom.kakao-sendTo-myself}",
+    @PostExchange(value = "${https://kapi.kakao.com/v2/api/talk/memo/default/send}",
```

Comment:
```suggestion
    @PostExchange(value = "/v2/api/talk/memo/default/send",
```
이렇게 하면 충분합니다.
이미 RestClient에서 host를 지정했습니다.